src/InstallerHosts/Elastic.InstallerHosts/Elasticsearch/Tasks/Immediate/ValidateArgumentsTask.cs

```diff
@@ -28,6 +29,11 @@ protected override bool ExecuteTask()
 				var validationFailures = ValidationFailures(failures);
 				throw new Exception("Cannot continue installation because of the following errors" + Environment.NewLine + validationFailures);
 			}
+
+			var installDir = this.Session.Get<string>("INSTALLDIR");
+			var version = this.InstallationModel.NoticeModel.CurrentVersion.ToString();
```

Comment:
Can this be part of `LocationsModelValidator` and the rule for the `InstallDir` property?